staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -152,31 +119,6 @@ func TestConditionalDelete(t *testing.T) {
 	storagetesting.RunTestConditionalDelete(ctx, t, store)
 }
 
-// The following set of Delete tests are testing the logic of adding `suggestion`
```

Comment:
I think we might want to keep this in the storagetesting package just for historical explanation?

Reply:
It's already there - we had two copies:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/testing/store_tests.go#L273

I'm removing only the one from here, keeping the other one where the tests are actually implemented now.

Reply:
Sounds good!